GeeksCoreLibrary/Modules/Databases/Services/MySqlDatabaseHelpersService.cs

```diff
@@ -757,5 +757,14 @@ private static (string DatabaseValue, bool unsigned) MySqlDbTypeToDatabaseValue(
 
             return (mysqlType, unsigned);
         }
+        
+        /// <inheritdoc />
+        public async Task OptimizeTablesAsync(params string[] tableNames)
+        {
+            if (tableNames.Any())
+            {
+                await databaseConnection.ExecuteAsync($"OPTIMIZE TABLE {String.Join(',', tableNames)}");
```

Comment:
Dit vind ik wel gevaarlijk. Je voegt geen backticks toe aan de tabelnamen en je voegt de parameter direct in de query toe. Dan heb je mogelijk SQL injection als deze functie ooit ergens op een manier wordt uitgevoerd waarbij je via front-end de tabelnaam kunt opgeven. Ook zal dit fout gaan wanneer er ooit een tabel is die bijvoorbeeld `order` heet, omdat dat een gereserveerd woord is in SQL.

Kun je deze code aanpassen naar zoiets?
```C#
await databaseConnection.ExecuteAsync($"OPTIMIZE TABLE {String.Join(',', tableNames.Select(t => $"`{t.ToMySqlSafeValue(false)}`"))}");
```

Reply:
De code is aangepast naar bovenstaande regel en getest. De gebruikte query heeft nu de backticks om de tabelnamen en de function om de naam SQL veilig te maken wordt aangeroepen. Het tweede kan wel een ongeldige query geven doordat de tabel niet gevonden kan worden maar aangezien de tabel toch niet kan bestaan is dit niet erg.